app/models/user.rb

```diff
@@ -1,14 +1,14 @@
 class User < ActiveRecord::Base
+  has_secure_password
   has_attached_file :picture, styles: {micro: '50x50',
-                                     thumb: '100x100',
-                                     small: '200x200',
-                                     medium: '300x300'
-                                   }
+                                       thumb: '100x100',
+                                       small: '200x200',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.